Source/SwiftLintFramework/Extensions/SwiftExpressionCallKind.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SwiftExpressionCallKind.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 11/12/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public enum SwiftExpressionCallKind: String {
```

Comment:
Can we make this `internal` since we'll probably want to move to SourceKitten in the near future?

FYI, all the "Expression" kinds are listed here: https://github.com/apple/swift/blob/swift-3.0.1-RELEASE/tools/SourceKit/lib/SwiftLang/SwiftLangSupport.cpp#L143-L147

At this moment, that's `call`, `argument`, `array`, `dictionary` and `object_literal`.

Since we already use some of those in existing SwiftLint rules, it'd be nice to rename this `SwiftExpressionKind`, update the other rules to use this, and move to SourceKittenFramework.